ecs-agent/netlib/platform/containerd_debug_linux.go

```diff
@@ -10,5 +10,9 @@ type containerdDebug struct {
 }
 
 func (c *containerdDebug) CreateDNSConfig(taskID string, netNS *tasknetworkconfig.NetworkNamespace) error {
-	return c.common.createDNSConfig(taskID, true, netNS)
+	// SC tasks will get DNS config from control plane
```

Comment:
nonblocking: can we have some context why the DNS config from customer can help for SC?

Reply:
can we have a unit test for this case?